.github/CODEOWNERS

```diff
@@ -10,6 +10,9 @@
 /packages/payment-integration-api @bigcommerce/team-checkout
 /packages/test-framework @bigcommerce/team-checkout
 /packages/workspace-tools @bigcommerce/team-checkout
+/packages/legacy-hoc @bigcommerce/team-checkout
+/packages/dom-utils @bigcommerce/team-checkout
+/packages/error-handling-utils @bigcommerce/team-checkout
```

Comment:
Perhaps we can sort the `* @bigcommerce/team-checkout` list alphabetically.

Reply:
@bc-peng Sorted.